app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -134,6 +134,14 @@ XrResult OpenXRInputSource::Initialize()
         RETURN_IF_XR_FAILED(OpenXRExtensions::sXrCreateHandTrackerEXT(mSession, &handTrackerInfo,
                                                                       &mHandTracker));
 
+#if defined(PICOXR)
+        // Pico's runtime does not advertise it but it does work.
+        mSupportsFBHandTrackingAim = true;
+#else
+        mSupportsFBHandTrackingAim = OpenXRExtensions::IsExtensionSupported(XR_FB_HAND_TRACKING_AIM_EXTENSION_NAME);
+#endif
+        VRB_LOG("OpenXR: using %s to compute hands aim", mSupportsFBHandTrackingAim ? "XR_FB_HAND_TRACKING_AIM" : "hand joints");
+
         if (OpenXRExtensions::IsExtensionSupported(XR_FB_HAND_TRACKING_MESH_EXTENSION_NAME) &&
```

Comment:
Just wanted to understand it well; although PICOXR supports the extension, despite not announcing it, we don't want it to go through this branch, right ?  And if that's the case, why ?

Reply:
You're right we want Pico to use the extension. The reason why we do that is because it provides a better and more stable aim than when using the joints of the hand.